index.html

```diff
@@ -156,5 +156,18 @@ <h4 class="connect">LET'S CONNECT</h4>
             </div>
         </div>
     </section>
+    <div id="class_background">
+        <section class="contactMe" id="contact_back">
+            <h2>Contact me</h2>
+            <p class="contact_info">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+            I'd love to help with it</p>
+            <form action="">
```

Comment:
- [ ] By convention, forms are supposed to get information from users and send it over to the server side as connected. So, this form needs action and method attributes to integrate with [formspree service](https://formspree.io/html).

Something like this:
```
<form action="https://formspree.io/f/{form_id}" method="POST">
```
_You will get an `id` by signing up for an account for [formspree service](https://formspree.io/html). Don't worry, it's free 😃